deploy-manage/monitor/autoops/autoops-stack-monitoring-comparison.md

```diff
@@ -0,0 +1,154 @@
+---
+applies_to:
+  deployment:
+    ess: all
+    ece: all
+    eck: all
+    self: all
+navigation_title: AutoOps and Stack Monitoring comparison
+---
+
+# AutoOps and Stack Monitoring comparison
+
+This page provides a detailed comparison of AutoOps and Stack Monitoring to help you decide which solution is better suited to your needs.
+
+## Review the main differences
+Review how both tools differ in their provisioning, setup, access, and investigative abilities.
```

Comment:
```suggestion
Review how these tools differ in their provisioning, set up procedure, method of access, and capabilities.
```
 - "both" reads a little oddly for me, since you couldn't really compare just one tool.
 - I'd remove "investigative", because otherwise "abilities" or "capabilities could be interpreted as applying to each noun in the list, i.e. "provisiioning ability", "setup abillity, etc."
 - "capabilities" strikes me as more fitting for something a machine does, and "abilities" for something a person does, but maybe that's just me. Just a guess. :-)